osu.Game/Screens/Edit/Editor.cs

```diff
@@ -337,6 +339,10 @@ private void load(OsuConfigManager config)
                                             new ToggleMenuItem(EditorStrings.AutoSeekOnPlacement)
                                             {
                                                 State = { BindTarget = editorAutoSeekOnPlacement },
+                                            },
+                                            new ToggleMenuItem(EditorStrings.LimitedDistanceSnap)
+                                            {
+                                                State = { BindTarget = editorLimitedDistanceSnap },
```

Comment:
One thing I'm not entirely sure about is this being in the core editor menu. Of all four official rulesets currently, two don't have (and never will have) a distance snap grid, the third (catch) is a _different_ sort of distance snap grid and this setting as implemented for osu! doesn't really make much sense there, and the fourth is osu! the ruleset. So this is a bit wonky.

If you're fine with this for now then probably so am I, but just voicing remaining concerns before I go clicking any buttons.

Reply:
It's pretty much a temporary thing yeah. Need to allow rulesets to populate menus or something at some point.